### DISCUSSION TITLE
Not compatible to sdk 33 when using jetpack

### DISCUSSION THREAD

**Discussion by greatfighter** [patrykandpatrick/vico#570] [Questions]
The library is not compatible to sdk 33 on my device.

**Comment by patrickmichalik** [comment]
Hello! Vico is compiled against the latest version of the Android SDK, which is 34. The same applies to many official libraries (including recent releases of Jetpack Compose). Please set `compileSdk` to 34. `targetSdk` can remain unchanged.
